core/src/test/scala/org/apache/spark/memory/StaticMemoryManagerSuite.scala

```diff
@@ -48,7 +48,10 @@ class StaticMemoryManagerSuite extends MemoryManagerSuite {
       conf.clone
         .set("spark.memory.fraction", "1")
         .set("spark.testing.memory", maxOnHeapExecutionMemory.toString)
-        .set("spark.memory.offHeap.size", maxOffHeapExecutionMemory.toString),
+        .set("spark.memory.offHeap.size",
+          if (maxOffHeapExecutionMemory != 0L) { maxOffHeapExecutionMemory.toString } else {
+            conf.get("spark.memory.offHeap.size", maxOffHeapExecutionMemory.toString)
```

Comment:
If `maxOffHeapExecutionMemory` is not explicitly specified like [this code](https://github.com/apache/spark/blob/master/core/src/test/scala/org/apache/spark/memory/MemoryManagerSuite.scala#L162)), we want to use the value in `"spark.memory.offHeap.size"` when `spark.memory.offHeap.enabled` is `true`.